### DISCUSSION TITLE
Commented-out namespace

### DISCUSSION THREAD

**Discussion by Macintosh-Fan** [jpb10/SolarCalculator#24] [General]
While browsing through the main header file, I noticed a namespace declaration is commented out. Why is that the case?
<img width="239" alt="image" src="https://user-images.githubusercontent.com/50642934/231008438-607b2365-ed63-4cbe-8ce8-75562d46066c.png">
<img width="139" alt="image" src="https://user-images.githubusercontent.com/50642934/231008456-51653e43-1f80-4d80-a36a-487fc3c9b399.png">

**Comment by jpb10** [comment]
I initially wasn't familiar enough with the Arduino library ecosystem to realize that it was necessary.

I commented it out to retain backwards compatibility. Feel free to uncomment if running into name clashes.

**Comment by Macintosh-Fan** [comment]
I see. Thanks!
